tools/ci/ci_agent.py

```diff
@@ -278,6 +312,68 @@ def _run_targets(targets, src, build_path, config):
     return result
 
 
+def _build_code_coverage(src, build_path, build_flags, c_flags, codecov_token):
+    result = {}
+    try:
+        target_result = result.setdefault("coverage", {})
+        target_build_result = target_result.setdefault("CodeCoverage", {})
+        out = _build_target("coverage", src, build_path, build_flags, c_flags)
+        log(out)
+        out = _run_cmd(f"gcovr -r . -x -o {build_path}/cobertura.xml")
```

Comment:
Does this need to be named cobertura?

Reply:
yes as we can use this file with cobertura as well